src/win/util.c

```diff
@@ -1397,6 +1397,16 @@ int uv_os_get_passwd(uv_passwd_t* pwd) {
 }
 
 
+int uv_os_environ(uv_env_item_t** envitems, int* count) {
```

Comment:
Just in case you're looking for some reference Windows code: https://github.com/nodejs/node/blob/f53a2414474c2b1aee07ab367415b32c20e50bd9/src/node_report.cc#L533-L552